int32 joint
float32 position
float32 speed
